wire-ios/Wire-iOS/Sources/UserInterface/GroupDetails/GroupParticipantsDetail/GroupParticipantsDetailViewModel.swift

```diff
@@ -139,7 +139,11 @@ final class GroupParticipantsDetailViewModel: NSObject, SearchHeaderViewControll
                 guard let conversation = conversation as? ZMConversation else { continue }
                 do {
                     let isE2EICertified = try await isUserE2EICertifiedUseCase.invoke(conversation: conversation, user: user)
-                    userStatuses[user.remoteIdentifier]?.isE2EICertified = isE2EICertified
+                    if userStatuses.keys.contains(user.remoteIdentifier) {
+                        userStatuses[user.remoteIdentifier]?.isE2EICertified = isE2EICertified
+                    } else {
+                        userStatuses[user.remoteIdentifier] = userStatuses[user.remoteIdentifier] ?? .init(user: user, isE2EICertified: isE2EICertified)
```

Comment:
I don't understand this line. The if before already makes sure that `userStatuses[user.remoteIdentifier]` is null, so why does it appear on the right side?

Reply:
Good catch. First I tried to not use an `if`, then after adding the `if` I missed it to fix the line.